cfpb/consumerfinance.gov
# Docker python upgrade 3.9

These changes to docker file, entry point, and magian.py files should now leet our docker instances use python3.9.9


---


## Removals

- no longer using the SCL way of installing python


## Changes

- pip calls now use python3 -m to call
- upgraded psycog2
- calls to python have been replaced with calls to python3


## How to test this PR

1. Pull the PR locally 
2. Fallow the instructions to set up docker if its your first time
3. run docker-compose up
4. if your python image is not rebuilt you might need to get rid of the old image
5. once docker compose has created the containers see if localhost:8000 is up and running


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
